nncf/experimental/openvino_native/graph/model_transformer.py

```diff
@@ -107,3 +113,17 @@ def _insert_outputs(model: ov.Model, outputs: List[ov.Output]) -> ov.Model:
             extra_model_outputs.append(result)
 
         return ov.Model(model_outputs + extra_model_outputs, params)
+
+    def _apply_node_removing_transformation(self, transformations: List[OVNodeRemovingCommand]) -> None:
```

Comment:
Does the function support removing a node with multiple input ports?

Reply:
If you mean FakeQuantizes - yes, it removes it. 
The thing is that we need a quantization algorithm (aka MinMax) to get the quantized model and test the command correctly. And this is why the test removes the original FP nodes with one input now.